dev-util/perf/perf-6.6.ebuild

```diff
@@ -209,6 +209,20 @@ perf_make() {
 	local arch=$(tc-arch-kernel)
 	local java_dir
 	use java && java_dir="${EPREFIX}/etc/java-config-2/current-system-vm"
+
+	# sync this with the whitelist in tools/perf/Makefile.config
```

Comment:
this is grim but not your fault at all, just the way of the kernel build system stuff..